meta-lmp-base/recipes-support/mfgtool-files/mfgtool-files_0.1.bb

```diff
@@ -28,7 +29,8 @@ PACKAGE_ARCH = "${MACHINE_ARCH}"
 
 do_compile() {
     sed -e 's/@@MACHINE@@/${MACHINE}/' ${S}/bootloader.uuu.in > bootloader.uuu
-    sed -e 's/@@MACHINE@@/${MACHINE}/' ${S}/full_image.uuu.in > full_image.uuu
+    sed -e 's/@@MACHINE@@/${MACHINE}/' ${S}/full_image.uuu.in > full_image.uuu.tmp
+    sed -e 's/@@MFGTOOL_FLASH_IMAGE@@/${MFGTOOL_FLASH_IMAGE}/' ${S}/full_image.uuu.tmp > full_image.uuu
```

Comment:
No need for 2 sed lines here, you can add another command with another -e after the first, that way we don't need to generate a tmp file.